client/src/core/client/admin/routes/Reports/ReportStatusMenu.css

```diff
@@ -0,0 +1,7 @@
+.statusLabel {
+  color: var(--palette-text-000);
```

Comment:
lol, forgot we had a `...-text-000`, always seemed like a funny naming convention... wouldn't that make it transparent? 🤣